frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/os-flavor.tsx

```diff
@@ -145,6 +148,22 @@ export const OSFlavor: React.FC<OSFlavorProps> = React.memo(
     );
     const baseImage = operatingSystemBaseImages.find((image) => image.id === os);
 
+    const numOfMountedDisks = cloneBaseDiskImage + mountWindowsGuestTools; // using boolean addition operator to count true
+    const mountedDisksHelpMsg = numOfMountedDisks > 0 && (
+      <Text className="kv-create-vm__input-text-help-msg">
+        View the mounted {pluralize(numOfMountedDisks, 'disk', 'disks', false)} in the{' '}
```

Comment:
we also have pluralize function in our package which is easier to use and does not depend on details-page file

Reply:
using our own pluralize :-)